gui-js/apps/minsky-electron/src/app/managers/CommandsManager.ts

```diff
@@ -1144,16 +1150,31 @@ export class CommandsManager {
         webContents.close();
       }
     });
+
+    // handler for updating progress bar
+    item.on('updated',(event,state)=>{
+      switch (state) {
+      case 'progressing':
+        if (!progress?.isCompleted())
+          progress.value=100*item.getReceivedBytes()/item.getTotalBytes();
+        break;
+      case 'interrupted':
+        //item.resume();
+        break;
+      }
+    });
   }
 
   // handler for downloading Minsky
   static downloadMinsky(event,item,webContents) {
     item.setSavePath(join(tmpdir(),item.getFilename()));
 
-    let progress=new ProgressBar({text:"Downloading Ravel application",value: 0, indeterminate:false, closeOnComplete: true,});
+    let progress=new ProgressBar({text:"Downloading Minsky application",value: 0, indeterminate:false, closeOnComplete: true,});
```

Comment:
We're actually rebadging this as "Ravel application", rather than Minsky. Purely for marketing reasons :).